flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/WindowOperator.java

```diff
@@ -264,7 +274,9 @@ public void dispose() {
 	@Override
 	@SuppressWarnings("unchecked")
 	public void processElement(StreamRecord<IN> element) throws Exception {
-		Collection<W> elementWindows = windowAssigner.assignWindows(element.getValue(), element.getTimestamp());
+
+		Collection<W> elementWindows = windowAssigner.
```

Comment:
Here I'm just being nitpicky but the code looks strange if broken into lines like this. To me
```
Collection<W> elementWindows =
    windowAssigner.assignWindows(element.getValue(), element.getTimestamp());
```
or
```
Collection<W> elementWindows = windowAssigner.assignWindows(
    element.getValue(),
    element.getTimestamp()); 
```

seem more elegant.